### DISCUSSION TITLE
Feature Request Login für SAE

### DISCUSSION THREAD

**Discussion by KingLuy** [camueller/SmartApplianceEnabler#495] [Allgemein]
Hallo Herr Müller,

ich hab den SAE soweit zum laufen gebracht. Cooles Projekt!!! Jetzt bin ich ja neugierig und greife auch gern von unterwegs auf mein System zu. (Benutze derzeit kein VPN) Daher habe ich den Port für den SAE ins Internet freigegeben. Da kam bei mir sofort die Frage auf ob es in Planung ist die SAE Weboberfläche mit eine Benutzerbereich zu versehen der eine erstmalige Anmeldung erfordert um auf die Weboberfläche zu gelangen?

Grüße King Luy

**Comment by camueller** [comment]
Das wird es nicht geben. Wer so etwas möchte, kann einen Reverse-Proxy davor schalten, der die Authentifizierung übernimmt.
